package.json

```diff
@@ -5,7 +5,7 @@
     "repository": "https://github.com/trezor/trezor-suite.git",
     "license": "SEE LICENSE IN LICENSE.md",
     "engines": {
-        "node": ">=12",
+        "node": ">=14",
```

Comment:
I'd keep minor and fix version as in build target `>=14.16.0`

Reply:
5166fc4